src/Reports/SiteSummary.php

```diff
@@ -41,10 +41,14 @@ public function sourceRecords($params)
 
     public function columns()
     {
-        return [
+        $columns = [
             'Summary' => 'Description',
             'Version' => 'Version',
         ];
+
+        $this->extend('updateColumns', $columns);
+
+        return $columns;
```

Comment:
Following on from previous comment (for `Package.php`)

I think I'm guilty of a lack of foresight, and that this is where the alterations to encapsulate the reports unique screen presentation should be.

```
public function columns()
{
    $columns = Package::create()->summaryFields();
    $unwantedColumns = array_flip(['Name', 'Description']);
    $columns = array_diff_key($columns, $unwantedColumns);
    $columns = array_merge(['Summary' => 'Description'], $columns);

    $this->extend('updateColumns', $columns);

    return $columns;
}
```